AlphaWallet.xcodeproj/project.pbxproj

```diff
@@ -6744,7 +6760,7 @@
 					"-framework",
 					SafariServices,
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = com.stormbird.alphawallet.AlphaWalletSafariExtension;
+				PRODUCT_BUNDLE_IDENTIFIER = com.westerlygoose.alphawallet.AlphaWalletSafariExtension;
```

Comment:
@eviltofu Please update your PR with removing your `bundle identifier`, the same for `DEVELOPMENT_TEAM`